documentation/src/docs/index.md

```diff
@@ -31,6 +31,8 @@ Read [Pravega Concepts](pravega-concepts.md) for more details.
  
 -   **Transaction Support:** A Pravega Transaction ensures that a set of events are written to a stream atomically. This is a key feature for distributed streaming applications requiring exactly-once guarantees on their output.
 
+- **Security and pluggable role based access control:** Pravega can be deployed by the administrators securely by enabling TLS for communications and can deploy their own implementation of role based access control plugin.   
```

Comment:
@fpj @arvindkandhare   Please note that Security Feature is added in the index.md!@